ush/rocoto/setup_expt_fcstonly.py

```diff
@@ -207,6 +207,23 @@ def edit_baseconfig():
       chgrp_rstprod = 'YES'
       chgrp_cmd = 'chgrp rstprod'
       hpssarch = 'YES'
+    elif machine == 'JET':
+      base_git = '/lfs4/HFIP/hfv3gfs/glopara/git'
+      base_svn = '/dev/null/global/save/glopara/svn/'
+      dmpdir = '/lfs4/HFIP/hfv3gfs/glopara/dump'
+      nwprod = '/lfs4/HFIP/hfv3gfs/glopara/nwpara'
+      comroot = '/lfs4/HFIP/hfv3gfs/glopara/com'
+      homedir = '/lfs4/HFIP/hfv3gfs/$USER'
+      stmp = '/lfs4/HFIP/hfv3gfs/${USER}/stmp'
+      ptmp = '/lfs4/HFIP/hfv3gfs/${USER}/ptmp'
+      noscrub = '$HOMEDIR'
+      account = 'hfv3gfs'
+      queue = 'batch'
+      queue_service = 'service'
+      partition_batch = "${PARTITION_BATCH:-'xjet'}"
```

Comment:
Please change to:
`partition_batch = 'xjet'`

Reply:
It can not be hard wired to xjet. JET can support many partitions.

Reply:
This sets the default that works best for users who don't know how to target another partitions and get's a partition correctly set in the resulting xml. Users can still change the partition in their respective config.base as needed (if they know what they are doing). Additional multi-partition support can be included in the full port work that will be done next. The support for different partitions will be explained by me when I announce this commit so don't worry about that. Please make the change I requested, thanks.